pkg/loadbalancer/config.go

```diff
@@ -305,6 +315,9 @@ func (def UserConfig) Flags(flags *pflag.FlagSet) {
 
 	flags.Bool(EnableHealthCheckNodePortName, def.EnableHealthCheckNodePort, "Enables a healthcheck nodePort server for NodePort services with 'healthCheckNodePort' being set")
 
+	flags.Bool(DropTrafficToVirtualIPsName, def.DropTrafficToVirtualIPs, "Drop traffic to virtual IPs (ClusterIP/LoadBalancer) on ports without services instead of forwarding (experimental)")
+	flags.MarkHidden(DropTrafficToVirtualIPsName)
```

Comment:
So we have marked this flag as experimental and hidden it. What is the path forward?

I think we should just expose the flag, add a helm value, and encourage people to use it. Shake out any issues that way, otherwise it will stay experimental and hidden forever to die of code rot.

Reply:
Another good step for gaining confidence in the feature is to add it to the verifier complexity tests which now can also do load time value permutations https://github.com/cilium/cilium/blob/main/pkg/datapath/loader/verifier_load_test.go.

And perhaps enabling this in some of our e2e / connectivity tests (assuming they do not assert on the specific behavior that changes).